manifests/s/ShiningLight/OpenSSL/1.0.0/ShiningLight.OpenSSL.installer.yaml

```diff
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: ShiningLight.OpenSSL
+PackageVersion: 1.0.0
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Commands:
+- openssl
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  Scope: machine
+  InstallerUrl: https://slproweb.com/download/Win64OpenSSL-1_1_1k.msi
+  InstallerSha256: C21E1B472C7777C7B2017FD335FC827976944BB1349731989C8D9E16D913519A
+  UpgradeBehavior: install
+  ProductCode: '{DE02BC89-4A1D-4B5D-A673-DE699BE1119F}'
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: msi
+  Scope: machine
+  InstallerUrl: https://slproweb.com/download/Win32OpenSSL_Light-1_1_1k.msi
```

Comment:
Hey @quhxl,

Is this the correct installerURL for this package? It seems to be pointing to ShiningLight.OpenSSLLight and not OpenSSL. Can you verify or make this change? I believe the correct url should be: https://slproweb.com/download/Win32OpenSSL-1_1_1k.msi

Thanks!

Reply:
Both urls got `404 Not Found` 👌